src/features/content-answers/body/ContentAnswersGridCard.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../../styles/variables';
+
+.ContentAnswersGridCard {
+    margin-top: 16px;
```

Comment:
yeah we do have spacing related variable but only 4px. should I create more such as `$bdl-grid-unit-4: 16px` instead of writing `4 * $bdl-grid-unit`
https://github.com/box/box-ui-elements/blob/65eb428746415e92abf83961c806f18a55de57f0/src/styles/constants/_layout.scss#L19-L20

Reply:
its fine to `$bdl-grid-unit * 4`, just make sure the variable is first, i have even seen some people have `variable * fraction` before